pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/ModelInterpreter.groovy

```diff
@@ -172,8 +172,7 @@ public class ModelInterpreter implements Serializable {
             } catch (MissingContextVariableException e) {
                 if (FilePath.class.equals(e.type) || Launcher.class.equals(e.type)) {
                     if (!agent.hasAgent()) {
-                        script.error("Attempted to execute a step that requires a node context while 'agent none' was specified. " +
-                                "Be sure to specify your own 'node { ... }' blocks when using 'agent none'.")
+                        script.error(Messages.ModelInterpreter_NoNodeContext())
```

Comment:
Not really necessary since this won't execute on a user request thread you'd most likely get the default language anyway.